hbase-server/src/main/java/org/apache/hadoop/hbase/monitoring/TaskMonitor.java

```diff
@@ -85,14 +85,22 @@ public static synchronized TaskMonitor get() {
   }
 
   public MonitoredTask createStatus(String description) {
-    return createStatus(description, false);
+    return createStatus(description, true);
   }
 
-  public MonitoredTask createStatus(String description, boolean ignore) {
-    return createStatus(description, ignore, false);
+  public MonitoredTask createStatus(String description, boolean track) {
+    return createStatus(description, track, false);
   }
 
-  public synchronized MonitoredTask createStatus(String description, boolean ignore,
+  /**
+   * Create a monitored task for users to inquire about the status
+   * @param description   description of the status
+   * @param track         whether to track(e.g. show/clear/expire) the task in the
+   *                      {@link TaskMonitor}
+   * @param enableJournal enable when the task contains some stage journals
+   * @return a monitored task
+   */
+  public synchronized MonitoredTask createStatus(String description, boolean track,
```

Comment:
since you inverted this boolean, we need to update all callers to invert whatever they passed previously. looks like there are only a handful

Reply:
What about keep the old behavior of MonitoredTask#createStatus? I changed the variable name to `ignoreSubTasksInTaskMonitor`. Thanks.

Reply:
Yes, that works, thanks!